pkg/question/question.go

```diff
@@ -1014,11 +1014,15 @@ func AskConfirmationWithInput(simpleConfig *config.SimpleInfo, detailedConfig *c
 		data = append(data, []string{cli.ResourceIamInstanceProfile, simpleConfig.IamInstanceProfile})
 	}
 
-	if simpleConfig.UserDataFilePath != "" {
-		data = append(data, []string{cli.ResourceUserDataFilePath, simpleConfig.UserDataFilePath})
+	if simpleConfig.BootScriptFilePath != "" {
+		data = append(data, []string{cli.ResourceBootScriptFilePath, simpleConfig.BootScriptFilePath})
 	}
 	if simpleConfig.UserTags != nil {
-		data = append(data, []string{cli.ResourceUserTags, strings.Join(simpleConfig.UserTags, "\n")})
+		var tags []string
+		for k := range simpleConfig.UserTags {
+			tags = append(tags, fmt.Sprintf("%s | %s", k, simpleConfig.UserTags[k]))
```

Comment:
format below `tag1|val1` without spaces. May be get rid of extra spaces for clarity? 

Or how about adding a header to the tags slice here like tags
```
var tags = []string{"KEY | VALUE"}
```

Reply:
* Removing the spaces from `tag1|val1` 
* Your suggestion is definitely clearer, but the way table is implemented and formatted now it'd be difficult to get such a view today
* In the mean time, I'll format it in the final confirmation to be cleaner and revisit when/if we refactor the table logic:
```
+--------------------------------------+--------------------------+
| Tag Specification(key|value)         | taggd|Bryan              |
|                                      | revd:|pdk                |
+--------------------------------------+--------------------------+

```